CopilotKit/packages/react-ui/src/components/chat/Input.tsx

```diff
@@ -62,11 +62,13 @@ export const Input = ({ inProgress, onSend, isVisible = false }: InputProps) =>
         value={text}
         onChange={(event) => setText(event.target.value)}
         onKeyDown={(event) => {
-          if (event.key === "Enter" && !event.shiftKey) {
-            event.preventDefault();
+           if (event.key === "Enter" && !event.shiftKey) {
+             event.preventDefault();
+           if (text.trim() !== "") { // Check if input is not empty
```

Comment:
```suggestion
           if (!sendDisabled ) { // Check if send is not disabled
```

Reply:
Thanks, I just saw this change!

Reply:
https://github.com/CopilotKit/CopilotKit/pull/1135

Reply:
okk